tests/pcond_test.py

```diff
@@ -0,0 +1,37 @@
+from honeybee_radiance_command.pcond import Pcond
+import pytest
+import honeybee_radiance_command._exception as exceptions
+
+
+def test_defaults():
+    pcond = Pcond()
+    assert pcond.command == 'pcond'
+    assert pcond.options.to_radiance() == ''
+    with pytest.raises(exceptions.MissingArgumentError):
+        pcond.to_radiance()
+
+
+def test_assignment():
+    pcond = Pcond()
+
+    pcond.input = 'image.hdr'
+    assert pcond.input == 'image.hdr'
+    assert pcond.to_radiance() == 'pcond image.hdr'
+    pcond.output = 'conditioned_image.hdr'
+    assert pcond.output == 'conditioned_image.hdr'
+    assert pcond.to_radiance() == 'pcond image.hdr conditioned_image.hdr'
+
+
+def test_stdin():
+    pcond = Pcond()
+    pcond.input = 'image.hdr'
+    pcond.output = 'conditioned_image.hdr'
+    assert pcond.to_radiance(stdin_input=True) == 'pcond conditioned_image.hdr'
```

Comment:
I just tested this and it doesn't seem to work. You can test it like this on Windows.

```type input.hdr | pcond conditioned.hdr```

Reply:
I had a hard time trying to understand how pipe_to works. Why do we have an argument named 'stdin_input' [here](https://github.com/ladybug-tools/honeybee-radiance-command/blob/master/honeybee_radiance_command/_command.py#L129) that apparently does not get used in the to_radiance method?

Reply:
I replied to this question in a different repository. stdin_input is not applicable to every command in Radiance. Some commands accept inputs from stdin and others don't. We don't have to use it for every command in Radiance. We should just use it where it makes sense. For the ones that don't accept stdin we can throw an error that this command doesn't accept inputs from stdin.